spark/build-images.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -e
+
+VERSION=3.5.1
+
+docker build -t spark-base:$VERSION ./spark-base
```

Comment:
Can you make them like prestodb/spark-base:$VERSION?
Also make sure the build-images.sh itself is executable.

Reply:
Fixed